chart/templates/configmap.yaml

```diff
@@ -81,14 +77,26 @@ data:
 
     redis_memory: "{{ .Values.redis_memory }}"
 
+    redis_storage: "{{ .Values.redis_storage }}"
 
     # crawler
     crawler_image: {{ .Values.crawler_image }}
     crawler_image_pull_policy: {{ .Values.crawler_pull_policy }}
 
-    crawler_cpu: "{{ mul .Values.crawler_cpu_per_browser .Values.crawler_browser_instances }}m"
+    crawler_cpu_base: "{{ .Values.crawler_cpu_base }}"
+    crawler_memory_base: "{{ .Values.crawler_memory_base }}"
+
+    crawler_extra_cpu_per_browser: "{{ .Values.crawler_extra_cpu_per_browser | default 0 }}"
+    crawler_extra_memory_per_browser: "{{ .Values.crawler_extra_memory_per_browser | default 0 }}"
```

Comment:
Do we want these defaults set to 0 or something higher?

Reply:
Maybe we could use the default values from values.yaml instead:

```
crawler_extra_cpu_per_browser: 300m

crawler_extra_memory_per_browser: 375Mi
```

Reply:
Hm, that might seem counterintuitive, or maybe it should just error if these settings are missing by not setting a default.
If you don't want an incremental value, maybe can just use crawl_cpu / crawl_memory directly